web-api/src/business/useCaseHelper/caseAssociation/updateCaseAndAssociations.ts

```diff
@@ -462,17 +419,22 @@ export const updateCaseAndAssociations = async ({
     .validate()
     .toRawObject();
 
-  const RELATED_CASE_OPERATIONS = [
+  let RELATED_CASE_OPERATIONS = [
     updateCaseDeadlines,
     updateCaseDocketEntries,
     updateCaseMessages,
-    updateCaseWorkItems,
-    updateCorrespondence,
     updateHearings,
     updateIrsPractitioners,
     updatePrivatePractitioners,
   ];
 
+  if (includeCorrespondenceAndWorkItems) {
+    RELATED_CASE_OPERATIONS = RELATED_CASE_OPERATIONS.concat([
+      updateCaseWorkItems,
+      updateCorrespondence,
+    ]);
+  }
```

Comment:
This is a temporary measure to improve efficiency as some things are in Dynamo and some in Postgres,